src/ext/spec/item.cr

```diff
@@ -0,0 +1,12 @@
+module Spec
+  module Item
+    def all_tags : Set(String)
```

Comment:
Tags on a spec don't include parent contexts (`context`, `describe`) so this monkey-patched method is to facilitate that.